src/main/java/study/calculate/Operator.java

```diff
@@ -1,25 +1,25 @@
-package study.operator;
+package study.calculate;
 
 import java.util.Arrays;
 import java.util.function.BiFunction;
 
 public enum Operator {
-    PLUS("+", (first, second) -> first + second),
-    MINUS("-", (first, second) -> first - second),
-    TIMES("*", (first, second) -> first * second),
-    DIVIDE("/", (first, second) -> first / second);
+    PLUS("+", IntNumber::plus),
+    MINUS("-", IntNumber::minus),
+    TIMES("*", IntNumber::times),
+    DIVIDE("/", IntNumber::divide);
 
     private static final String INVALID_OPERATOR_EXCEPTION_MESSAGE = "not supported operator.";
 
     private final String symbol;
-    private BiFunction<Integer, Integer, Integer> operate;
+    private BiFunction<IntNumber, IntNumber, IntNumber> operate;
```

Comment:
이렇게 입력 인자 2개와 출력 인자 1개가 모두 동일한 타입일 경우에는 BiFunction보다는 
BinaryOperator가 편합니다 ㅎㅎ
```suggestion
    private BinaryOperator<IntNumbe> operate;
```

Reply:
@catsbi 
요런 functional interface도 있군요.. 감사합니다 👍